stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ChannelUtils.kt

```diff
@@ -91,12 +91,22 @@ public fun Channel.getLastMessagePreviewText(
                     )
                 }
 
-                val attachmentText = message.attachments
-                    .takeIf { it.isNotEmpty() }
-                    ?.mapNotNull { attachment ->
-                        attachment.title ?: attachment.name
-                    }
-                    ?.joinToString()
+                val attachmentText: String? = if (message.attachments.isNotEmpty()) {
+                    val textFormatter = ChatTheme.attachmentFactories
+                        .firstOrNull { it.canHandle(message.attachments) }
+                        ?.textFormatter
+
+                    message.attachments
+                        .takeIf { it.isNotEmpty() }
```

Comment:
I think we already checked `message.attachments.isNotEmpty()` in the above if-statement.

Reply:
Nice catch